examples/graphql-cache-invalidation/graphql_cache_invalidation.md

```diff
@@ -0,0 +1,23 @@
+# GraphQL service - Cache Invalidation
+
+The Ballerina `graphql` module provides functionality for cache invalidation. The `evictCache(resourcePath: string)` and `invalidateAll()` APIs in the `graphql:Context` can be used to invalidate caches in a `graphql:Service`. To invalidate a specific cache in a `resource` function, the `evictCache(resourcePath: string)` API requires the developer to specify the name of the resource. On the other hand, the `invalidateAll()` API invalidates all caches within the service.
```

Comment:
```suggestion
The Ballerina `graphql` module provides functionality for cache invalidation. The `invalidate(resourcePath: string)` and `invalidateAll()` APIs in the `graphql:Context` can be used to invalidate caches in a GraphQL service. To invalidate a specific cache in a `resource` function, the `invalidate(resourcePath: string)` API requires the developer to specify the name of the resource. On the other hand, the `invalidateAll()` API invalidates all caches within the service.
```
We have renamed the `evictCache()` to `invalidate()`. Check other places as well.

Reply:
Fixed in [1278ea3](https://github.com/ballerina-platform/ballerina-distribution/pull/5188/commits/1278ea379a55bf994c3f7d6de626a2bf60aba971)